projects/plugins/jetpack/extensions/blocks/podcast-player/api.js

```diff
@@ -9,12 +9,12 @@ import { PODCAST_FEED, EMBED_BLOCK } from './constants';
 import apiFetch from '@wordpress/api-fetch';
 import { addQueryArgs } from '@wordpress/url';
 
-export const fetchPodcastFeed = async url => {
+export const fetchPodcastFeed = async ( { url, guids } ) => {
 	// First try calling our endpoint for Podcast parsing.
 	let feedData, feedError;
 	try {
 		feedData = await apiFetch( {
-			path: addQueryArgs( '/wpcom/v2/podcast-player', { url } ),
+			path: addQueryArgs( '/wpcom/v2/podcast-player', { url, guids } ),
```

Comment:
_minor:_ Can we add the `guids` to the request query parameters only when it isn't empty? Something like the folloging (no checked):
```suggestion
			const queryParams = { url };
			if ( guids.length ) {
				queryParams.guids = guids;
			}
			path: addQueryArgs( '/wpcom/v2/podcast-player', queryParams ),
```